src/build.py

```diff
@@ -1233,15 +1241,15 @@ def Summary(repos):
       buildbot.Link('lkgr.json', cloud.Upload(info_file, lkgr_file))
 
 
-def AllBuilds(use_asm=False):
+def AllBuilds(use_asm=False, run_tests=False):
```

Comment:
It's way more readable to use a property name instead of a bunch of boolean arguments. 🤔 
Having everything in options also prevents random option-like things from coming to life when really they should be defaulted options (instead of e.g. defaulted function parameters).